tests/dummy/app/routes/employee.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class extends Route {
+  @service store;
+
+  model(params) {
+    return this.store
+      .findRecord('employee', params.employee_id)
+      .catch(() => null);
```

Comment:
later ember versions don't have implicit model hook that makes use of ember data store, so write it explicitly